.travis.yml

```diff
@@ -11,15 +11,14 @@ env:
       secure: IHUYMK2spxorl9lUeAbAfT6btuP2qRT615bUnEuUDgYrXf9y1CdQprYWJaygou/+6aWmLL0NnxYSpOxi40bHgMKeUQnTjXVbkkqQ1Tml3cMSsjkBrx7CNUygHvDDzCQCEC6m9uZjUKMZAVzVSWlOQhSMKR7MtdsSvMCrKIgA2pM=
 
 before_install:
+  - if [ $TRAVIS_PULL_REQUEST == "false" ]; then git clone https://github.com/$TRAVIS_REPO_SLUG.git $TRAVIS_REPO_SLUG && cd $TRAVIS_REPO_SLUG && git checkout -qf $TRAVIS_BRANCH; fi
```

Comment:
Ok, so for non-PR builds we're just checking out the current branch, and for PR's we're checking out the `$TRAVIS_COMMIT`, which I'm assuming is the temporary merge commit they make. How does this differ from the default behavior? Or is this just so you can run the npm commands within the correct commit to account for potential versioning changes?